### ISSUE TITLE
Please do a nuget release for 2.0.0-beta1

### ISSUE THREAD

**Issue by kinetiq** [toddams/RazorLight#228]
Could you please do a nuget release for 2.0.0-beta1? As you may know, you can do the release and mark it as prerelease so that it only shows up on for people are looking for it and willing to take a risk.

I've been told to get rid of PreserveCompilationContex / MvcRazorCompileOnPublish since it's having a negative effect on initial load time after deploy, and rendering from string would be a fantastic way to do that. So it's this or switch to a different tool for rendering emails. 

Thanks for the great project!

**Comment by kinetiq** [comment]
I hope @toddams  will weigh in on that, it's such a popular project.

What's the best alternative? I see that there's now a .NET Core version of RazorEngine in this fork: https://github.com/hermanho/RazorEngine/tree/updateToNetCore

**Comment by toddams** [comment]
Most of the critical bugs are fixed and changes are already in master.
Project is not abandoned, but is temporary frozen because of my real life problems. Most of them are solved so I expect to deliver a new release by the end of this month. 

I had a small hope on contributions from the community, but seems like it is not the case. Either should I document it better, or encourage you to help me out with this project :)

**Comment by kinetiq** [comment]
@toddams Thanks for the response. It's a great project! It does seem like it would take some work to get involved in it however.

**Comment by Aracturat** [comment]
@toddams Can we hope for new release by the end of october?

**Comment by kinetiq** [comment]
@toddams  Just checking in on this, as October is over. Thanks man.

**Comment by molinch** [comment]
@toddams Could we get a new Nuget release as this would fix a couple of issues?
Thanks a lot!

**Comment by wakuflair** [comment]
@toddams
Sorry but we're waiting for a new release.

**Comment by h0useRus** [comment]
@toddams is this project still supported?

**Comment by kinetiq** [comment]
Wow, I see that you actually did a nuget release for 2.0.0-beta1 but you didn't update this ticket. That is extremely unfortunate... I only noticed because I removed the package. 

Now I'm using this: https://github.com/aspnet/Entropy/blob/master/samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs

Just have to register  services.AddTransient<RazorViewToStringRenderer>(); and it seems to work like a charm:

var text = RazorRenderer.RenderViewToStringAsync<ActivateRespondentVM>("/Controllers/EmailTemplates/Views/ActivateRespondent.cshtml", mode);

...Where ActivateRespondentVM is the type of my model.

I'm sure the people who maintain this project must be aware of this. Is there some downside to this approach? It seems really easy and reliable, and works with .cshtml files that are compiled into the DLL.
